From 05cffcda932b0914d78a08891c13e61593a6ce02 Mon Sep 17 00:00:00 2001
From: Alexander Bersenev <bay@hackerdom.ru>
Date: Mon, 9 Jun 2014 00:08:13 +0600
Subject: [PATCH] ARM: sunxi: Enable IR controller on cubieboard 2 and
 cubietruck in dts

This patch enables two IR devices in dts:
- One IR device physically found on Cubieboard 2
- One IR device physically found on Cubietruck

Signed-off-by: Alexander Bersenev <bay@hackerdom.ru>
Signed-off-by: Alexsey Shestacov <wingrime@linux-sunxi.org>
---
 arch/arm/boot/dts/sun7i-a20-bananapi.dts | 6 ++++++
 arch/arm/boot/dts/sun7i-a20-cubietruck.dts  | 6 ++++++
 2 files changed, 12 insertions(+)

--- a/arch/arm/boot/dts/sun7i-a20-bananapi.dts
+++ b/arch/arm/boot/dts/sun7i-a20-bananapi.dts
@@ -103,6 +103,12 @@
 			status = "okay";
 		};
 
+		ir0: ir@01c21800 {
+			pinctrl-names = "default";
+			pinctrl-0 = <&ir0_pins_a>;
+			status = "okay";
+		};
+
 		uart0: serial@01c28000 {
 			pinctrl-names = "default";
 			pinctrl-0 = <&uart0_pins_a>;
